### PR TITLE
Add ST logo

### LINKED ISSUE #13: add ST branding
We still need to add the Social Tables logo and any other branding to give it that "ST feel"

### PR DESCRIPTION
**Pull request by danmactough**
resolves #13

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const Header = ({children, style}) => {
 		<AppBar 
 			style={style}
 			title={children}
+			titleStyle={ { position: "absolute", right: "10px" } }
```
Comment (rovolution) [inline src/components/header.js]: is there a way we can link the Logo to www.socialtables.com?

Comment (danmactough) [inline src/components/header.js]: The logo?
image

Sure. Do you like the placement of the logo and the title?

Comment (rovolution) [inline src/components/header.js]: yup looks solid!

Comment (danmactough) [inline src/components/header.js]: Seems like the material-ui convention is to put the text [title on the left and the icon on the right](http://www.material-ui.com/#/components/app-bar), so I swapped them.

image

**Comment by danmactough** [comment]
linked the logo to www via 5ebf5f6

**Pull request merged**